src/pages/About/AboutSection.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+class AboutSection extends Component {
```

Comment:
Not sure if a separate component is needed for this about section, there's really not much styling here. It's just two lines that can well be in the main About component map block.

Reply:
👍 
Fixed.

Reply:
Thanks 👍